src/storyblok-auth-api/handle-requests/openidClient.ts

```diff
@@ -6,25 +6,28 @@ import { getRegionBaseUrl, Region } from '@storyblok/region-helper'
 export type CreateOpenIdClient = (
   params: Pick<
     AuthHandlerParams,
-    'clientId' | 'clientSecret' | 'baseUrl' | 'endpointPrefix'
+    | 'clientId'
+    | 'clientSecret'
+    | 'baseUrl'
+    | 'endpointPrefix'
+    | 'storyblokApiBaseUrl'
   >,
   region?: Region,
 ) => BaseClient
 
 export const openidClient: CreateOpenIdClient = (params, region) => {
-  const { clientId, clientSecret } = params
+  const defaultEndpoint = region
+    ? `${getRegionBaseUrl(region)}`
+    : 'https://app.storyblok.com'
+
+  const { clientId, clientSecret, storyblokApiBaseUrl = undefined } = params
+  const oauthEndpoint = storyblokApiBaseUrl ?? defaultEndpoint
+
   const { Client } = new Issuer({
     issuer: 'storyblok',
-    // TODO: at this point there is no region && the subdomains do not have the /oauth/authorize endpoint working at the moment that is why this endpoint is initially requested
-    authorization_endpoint: `https://app.storyblok.com/oauth/authorize`,
-    token_endpoint:
-      typeof region !== 'undefined'
-        ? `${getRegionBaseUrl(region)}/oauth/token`
-        : undefined,
-    userinfo_endpoint:
-      typeof region !== 'undefined'
-        ? `${getRegionBaseUrl(region)}/oauth/user_info`
-        : undefined,
+    authorization_endpoint: `${oauthEndpoint}/oauth/authorize`,
```

Comment:
Just for me to understand, this line means that now the comment is no longer true, and not only `https://app.storyblok.com` has a response for `/oauth/authorize`, but any region-based URL? cc @johannes-lindgren

Reply:
Good point: the comment is still true

Maybe it does not need to be a TODO though

Let's add it back